code/src/archivedtests/pcgen/inttest/game_3e/pcGenGUIBrdJoeTest.java

```diff
@@ -1,44 +0,0 @@
-package pcgen.inttest.game_3e;
```

Comment:
This would be better as a move rather than as a delete and recreate. That way the history is preserved.